MobiusCore/Source/BackwardsCompatibility.swift

```diff
@@ -109,3 +109,29 @@ public extension EffectRouterBuilder {
         return addConnectable(connectable, predicate: action.canAccept)
     }
 }
+
+public extension Mobius {
+    /// Create a `Builder` to help you configure a `MobiusLoop ` before starting it.
+    ///
+    /// The builder is immutable. When setting various properties, a new instance of a builder will be returned.
+    /// It is therefore recommended to chain the loop configuration functions
+    ///
+    /// Once done configuring the loop you can start the loop using `start(from:)`.
+    ///
+    /// - Parameters:
+    ///   - update: the `Update` function of the loop
+    ///   - effectHandler: an instance conforming to the `ConnectableProtocol`. Will be used to handle effects by the loop
+    /// - Returns: a `Builder` instance that you can further configure before starting the loop
+    @available(*, deprecated, message: "Construct the loop with an `EffectHandler<Effect, Event>` instead")
```

Comment:
I’m confused. The way to combine `EffectHandler`s is currently still `EffectRouterBuilder`, which produces a `Connectable` rather than an `EffectHandler`.

Reply:
You are completely right. This should not be deprecated.